README.md

```diff
@@ -80,33 +76,23 @@ git clone https://github.com/celo-org/simple-celo-node.git
 cd simple-celo-node
 ```
 
-### Copy .env.example to .env
-
-Make a copy of `.env.example` named `.env`.
-
-```sh
-cp .env.example .env
-```
-
-Open `.env` with your editor of choice
+### Copy network env to .env
 
-### Mandatory configurations
+Copy the desired network environment file to `.env`.
 
-The default configuration can be used as is for running an Alfajores snap sync node.
+E.G. to run a node on Alfajores:
 
-If you want to run a Baklava node instead, set
-```
-NETWORK_NAME=baklava
-# Don't use these while baklava L1 is still running
-# HEALTHCHECK__REFERENCE_RPC_PROVIDER=https://baklava-forno.celo-testnet.org
-# OP_GETH__HISTORICAL_RPC=https://baklava-forno.celo-testnet.org
-# Use these instead:
-HEALTHCHECK__REFERENCE_RPC_PROVIDER=https://baklava-forno.baklava.celo-testnet.org
-OP_GETH__HISTORICAL_RPC=https://baklava-forno.baklava.celo-testnet.org
+```sh
+cp alfajores.env .env
 ```
+The `.env` file is ready to use and is configured for snap sync and non-archive mode. If you would like to customise
+your node further see [Optional configurations](#optional-configurations).
 
 ### Optional configurations
 
+* **NODE_TYPE** - Choose the type of node you want to run:
+    * `full` (Full node) - A Full node contains a few recent blocks without historical states.
+    * `archive` (Archive node) - An Archive node stores the complete history of the blockchain, including historical states.
```

Comment:
Do we support archive nodes? I didn't know that works with snap sync (which is the only sync mode supported according to the beginning of the README)

Reply:
Archive will work, enabling archive mode basically means we don't clear old state data, so it's just turning off the cleanup operation.

So with snap sync you will only have state from the block at which your snap sync completed.